javascript/javascript1/week2/index.js

```diff
@@ -87,4 +87,46 @@ function whatToWear(temperature) {
 
 whatToWear(18);
 whatToWear(2);
-whatToWear(30);
\ No newline at end of file
+whatToWear(30);
+
+
+//===================================================================================================================
+// Student Manager
+
+
+
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+    if (!studentName) {
+        return;
+    }
+    if (studentName === "queen") {
+        class07Students.push(studentName);
+    }
+    if (class07Students.length >= 6) {
+        console.log("Cannot add more students to class 07");
+    } else if (class07Students.includes(studentName)) {
+        console.log(`Student ${studentName} is already in the class.`);
```

Comment:
This condition should be the first here. Otherwise you will add multiple 'queen's in the array, right? So first we check if the new student is a duplicate, even 'queen' herself :)